handlers/geolocation/geolocation.go

```diff
@@ -109,7 +112,17 @@ func (c *GeolocationHandlersCollection) RedirectHandler() httprouter.Handle {
 			newURL.Scheme = protocol(r)
 			newURL.Host = nodeHost
 			http.Redirect(w, r, newURL.String(), http.StatusTemporaryRedirect)
-			glog.Infof("NodeHost redirect host=%s nodeHost=%s from=%s to=%s, lat=%s, lon=%s", host, nodeHost, r.URL, newURL, lat, lon)
+			jsonRedirectInfo, _ := json.Marshal(map[string]interface{}{
+				"redirect-type": "closest-node",
+				"host":          host,
+				"node-host":     nodeHost,
+				"playbackID":    playbackID,
+				"from":          r.URL.String(),
+				"to":            newURL.String(),
+				"lat":           lat,
+				"lon":           lon,
+			})
+			glog.Infof(string(jsonRedirectInfo))
```

Comment:
using the earlier pattern of `key1=value1 key2=value2` makes it easier on loki to parse, as you can simply use `| logfmt` and get access to all labels/keys for further filtering.